java/src/com/github/bazel_contrib/contrib_rules_jvm/junit5/TestData.java

```diff
@@ -16,9 +16,11 @@ class TestData {
   private final TestIdentifier id;
   private final List<ReportEntry> reportEntries = Collections.synchronizedList(new ArrayList<>());
   private Instant started = Instant.now();
-  private Instant finished = Instant.now();
+  // Commented out to avoid pulling in the dependency, but present for documentation purposes.
+  // @Nullable
+  private Instant finished = null;
   private String reason;
-  private TestExecutionResult result;
+  TestExecutionResult result;
```

Comment:
Why change the visibility? If this needs to be package private for some reason, can we document why?